src/lib.rs

```diff
@@ -473,20 +476,55 @@ impl Account {
     ) -> Result<(Account, AccountCredentials), Error> {
         Self::create_inner(
             account,
+            Key::generate()?,
             external_account,
             Client::new(server_url, http).await?,
             server_url,
         )
         .await
     }
 
+    /// Load a new account by private key, with a default or custom HTTP client
+    ///
+    /// https://www.rfc-editor.org/rfc/rfc8555#section-7.3.1
+    ///
+    /// The returned [`AccountCredentials`] can be serialized and stored for later use.
+    /// Use [`Account::from_credentials()`] to restore the account from the credentials.
+    pub async fn load(
+        private_key_pkcs8_der: &[u8],
+        server_url: &str,
+        http: Option<Box<dyn HttpClient>>,
+    ) -> Result<(Account, AccountCredentials), Error> {
+        let client = match http {
+            Some(http) => Client::new(server_url, http).await?,
+            None => Client::new(server_url, Box::new(DefaultClient::try_new()?)).await?,
+        };
+
+        let key = Key::from_pkcs8_der(private_key_pkcs8_der)?;
+        let pkcs8 = key.to_pkcs8_der()?;
+        let ignored_account = NewAccount {
+            only_return_existing: true,
+            contact: &[],
+            terms_of_service_agreed: true,
+        };
+
+        Self::create_inner(
+            &ignored_account, // This field is ignored as per rfc8555 7.3.1
+            (key, pkcs8),
+            None,             // This field is ignored as per rfc8555 7.3.1
```

Comment:
These comments don't quite make sense in their current form.

Reply:
The first comment is there to explain why a dummy ignored_account is created and why sending that as part of the request is okay. The second one might need to be removed - agreed.